### ISSUE TITLE
Could we get a more "Traditional" implementation of FSR and also NIS?

### ISSUE THREAD

**Issue by Alovon12** [yuzu-emu/yuzu#7921]
By "Traditional" I moreso mean things like

FSR Ultra Quality (1.3x)
FSR Quality (1.5x)
FSR Balanced (1.7x)
FSR Performance (2x)

As options that are applied on top of the Native resolution (0.5x, 0.75x, 1x, 2x)
As a way to get around the glitchiness reported when testing an option like 1.5x.

So instead of forcing the game to natively render at 1.5x, for example, someone could just select FSR 1.5x (What it could likely be called in the options) and it would effectively be 1440p if a game is 1080p locked at 1x, or 1080p if a game is locked at 720p (EX: XBC2).

The same things generally apply for NIS, I just feel adding NIS as an option would be nice as some games take better to NIS versus FSR due to differences in how they upscale and NIS seemingly working a bit better at reconstructing <1080p detail versus FSR, at least if the sharpening filter isn't cranked.

And on that note, exposing the sharpening filter would be nice too,

Also, I was told SMAA was coming?

**Comment by Alovon12** [comment]
This would also be upwards expandable too for games that have Resolution caps below 1080p, so if a game caps at 720p for example, 2x renders at 1440p.

So FSR Performance would be required to force the image up to a true pixel count of 4K

So a more traditional implementation would be scalable in both directions

**Comment by Skyrimer3d** [comment]
Yeah I'd really like to see this too, for some games 2x would be really useful for devices like Steam Deck that could get an extra boost in terms of performance.
